libs/labelbox/src/labelbox/schema/dataset.py

```diff
@@ -288,10 +290,9 @@ def create_data_rows(self,
         string_items = [item for item in items if isinstance(item, str)]
         dict_items = [item for item in items if isinstance(item, dict)]
         dict_string_items = []
-
         if len(string_items) > 0:
             dict_string_items = self._build_from_local_paths(string_items)
-        specs = DataRowUpsertItem.build(self.uid,
+        specs = DataRowCreateItem.build(self.uid,
```

Comment:
I thought the point was to maximize use of upsert ... was this on purpose or just for this use case?

Reply:
Yes we are using upsert, and this change just captures difference in data formatting and validation between create and upsert on the sdk side due to a number of edge cases for create